cylc/flow/cfgspec/workflow.py

```diff
@@ -1228,33 +1503,39 @@ def get_script_common_text(this: str, example: Optional[str] = None):
                     specified message rather than achieve a state.
                 ''')
                 Conf('run-dir', VDR.V_STRING, desc='''
+                    Specify the location of the top level cylc run directory.
+
                     For your own workflows the run database location is
                     determined by your site/user config. For other workflows,
```

Comment:
I'm not confident there is no need to set this, but the original description suggests it. Better test it out though
```suggestion
                    For your own workflows, there is no need to set this as it
                    is always `~/cylc-run/`. But for other workflows,
```